Kinvey/Kinvey/FileStore.swift

```diff
@@ -112,7 +112,7 @@ open class FileStore<FileType: File> {
         imageRepresentation: ImageRepresentation = .png,
         ttl: TTL? = nil,
         completionHandler: FileCompletionHandler? = nil
-    ) -> AnyRequest<Any> {
+    ) -> AnyRequest<Result<FileType, Swift.Error>> {
```

Comment:
Needs a doc change? If so, please open a devcenter PR.

Reply:
we don't have docs for sync / wait calls yet. I will create a ticket for that.